Cargo.toml

```diff
@@ -22,6 +22,7 @@ is-it-maintained-open-issues = { repository = "enarx/sgx" }
 
 [features]
 rcrypto = ["rand", "rsa", "sha2", "num-integer", "num-traits"]
+ioctls = []
```

Comment:
This is actually incorrect, since the `ioctls` feature requires `iocuddle`. It should be:

```
ioctls = ["iocuddle"]
```

However, I think we should just drop the separate `ioctls` feature flag. So just remove this line.